modules/builds-use-custom-builder-image.adoc

```diff
@@ -71,13 +68,9 @@ $ oc create -f imagestream.yaml
 
 . Run your custom build:
 +
+[source,terminal]
```

Comment:
Did you miss the oc commands on lines 66 and 49?

Reply:
Yep, fixed. Thanks!